vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/core/LayoutableShadowNode.cpp

```diff
@@ -27,31 +27,31 @@ static LayoutableSmallVector<Rect> calculateTransformedFrames(
   auto transformedFrames = LayoutableSmallVector<Rect>{size};
   auto transformation = Transform::Identity();
 
-  for (size_t i = size - 1; i >= 0; --i) { // changed from int to size_t, make PR upstream
+  for (size_t i = size; i > 0; --i) { // use size_t instead of int, and modify loop to avoid underflow, make PR upstream (Note changes to all uses of i in loop)
     auto currentShadowNode =
-        traitCast<LayoutableShadowNode const *>(shadowNodeList.at(i));
+        traitCast<LayoutableShadowNode const *>(shadowNodeList.at(i - 1));
     auto currentFrame = currentShadowNode->getLayoutMetrics().frame;
 
     if (policy.includeTransform) {
       if (Transform::isVerticalInversion(transformation)) {
         auto parentShadowNode =
-            traitCast<LayoutableShadowNode const *>(shadowNodeList.at(i + 1));
+            traitCast<LayoutableShadowNode const *>(shadowNodeList.at(i));
```

Comment:
what guarantees that this doesn't run on the first iteration?

Reply:
Did that change?  Couldn't it have run on the first iteration in core's code too?

Reply:
yes broken in both it seems, we should fix it?

Reply:
I'm just trying to keep the logic the same as core, while fixing the size_t int mismatch that causes our builds to break.  -- We should probably try to upstream this quickly though, since this offsets the meaning of i by one, which is likely to cause issues if they make additional changes to this file.